static/app/components/workflowEngine/layout/detail.tsx

```diff
@@ -20,12 +24,21 @@ export interface WorkflowEngineDetailLayoutProps {
  */
 function DetailLayout({children}: WorkflowEngineDetailLayoutProps) {
   const title = useDocumentTitle();
+  const organization = useOrganization({allowNull: false});
+  const project = useProjectFromSlug({organization, projectSlug: 'javascript'});
```

Comment:
Do we need to add a `projectSlug` prop to `DetailLayout`?

Ideally could we pass a partial `Project` to `DetailLayout` with the project slug and platform so that we don't need to query with `useProjectFromSlug`?

Reply:
Actually was in the middle of refactoring this right now! Ideally we'll get the project slug and platform from the backend and can skip the additional network request